Amazon.Ion.ObjectMapper.Test/TestObjects.cs

```diff
@@ -369,6 +373,30 @@ public override string ToString()
         }
     }
 
+    public class Desk
+    {
+        public int width;
```

Comment:
Why public?

Reply:
No special reason as the access modifiers don't influence the logic but I changed it to `internal` since that is more idiomatic for fields (`private` resulted in build problems when initializing the test Desk's width).